docs/maintenance/eslint-10-plugin-compatibility.md

```
@@ -0,0 +1,90 @@
+- `eslint-config-next/core-web-vitals`
+- `typescript-eslint` recommended type-checked and stylistic type-checked configs
+- `@tanstack/eslint-plugin-query` flat recommended config
+- Local plugin registrations for `@typescript-eslint`, `drizzle`, `import`, `react-hooks`, and `simple-import-sort`
+
+## Compatibility Table
+
+| Package | Current Sanbi version | Latest checked | ESLint 10 peer support | Notes |
+| --- | --- | --- | --- | --- |
+| `eslint` | `9.39.1` | `10.4.1` | N/A | Target major for the future migration. Latest package lists `jiti` as a peer. |
+| `eslint-config-next` | `16.2.6` | `16.2.7` | Declares `eslint: >=9.0.0` | Top-level peer range allows ESLint 10, but its dependency chain still includes plugins that do not currently declare ESLint 10 support. |
+| `typescript-eslint` | `8.59.4` | `8.60.1` | Yes: `^8.57.0 || ^9.0.0 || ^10.0.0` | Likely compatible after upgrading to `8.60.1`. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 9; Too many cells, extra data will be missing

(MD056, table-column-count)